src/card/card-media.jsx

```diff
@@ -47,6 +47,7 @@ const CardMedia = React.createClass({
         verticalAlign: 'top',
         maxWidth: '100%',
         minWidth: '100%',
+        width:'100%'
```

Comment:
yah, it looks like you can get away with just width: '100%' without max and min across all the major browsers